cmd/celestia-appd/cmd/root.go

```diff
@@ -34,7 +34,11 @@ const (
 	UpgradeHeightFlag = "v2-upgrade-height"
 
 	// TimeoutCommit is a flag that can be used to override the timeout_commit.
+	// Deprecated: Use BlockTimeFlag instead.
```

Comment:
```suggestion
	// Deprecated: Use DelayedPrecommitTimeoutFlag instead.
```

Reply:
fixed in https://github.com/celestiaorg/celestia-app/pull/5639/commits/1c8cd007ebd004eb449f2385693a4d550f2049f7